frameworks/elastic/tests/test_sanity.py

```diff
@@ -1,5 +1,3 @@
-import logging
```

Comment:
Yes, I saw another PR (I forget whose) that tried to remove `from tests.config import *` from all the shakedown files. Not sure what became of it.